### PR TITLE
Completed ML Labs

### PR DESCRIPTION
**Pull request by theckang**
Completed ML Labs.  Run the 'Feature Innovation with Serverless' setup to run these labs: https://github.com/RedHatGov/serverless-workshop-code

Users will:
* Deploy prediction service using OpenShift Serverless
* Debug the broken code in CodeReady Workspace and update with new revisions
* Integrate Twilio and Lifeline

One outstanding TODO:
* Image when Lifeline submits a help request to ER demo (not hooked up yet)

### CODE REVIEW

File: workshop/content/lab3.1_ml_overview.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Overview
+
+In this lab, you will add a new machine learning capability to the Emergency Response Demo using OpenShift Serverless.
+
+As you explored in the previous lab, an incident user must register to the Emergency Response application in order to create an incident for help.  This isn't the best user experience; in the middle of a disaster, people need the ability to ask for help quickly and efficiently.
+
+Let's create a new user experience.  A person texts a phone number to ask for help by sending details of the disaster they are experiencing.  On the backend, the system verifies if the message describes a legitimate disaster by using a machine learning Natural Language Processing (NLP) model.  If verification succeeds, the system replies to the user with a link to a mobile website to ask for help.  If it does not, the system asks for more details.  We will call this the **NLP Prediction Service** going forward.
+
+The backend for this service doesn't need to be running until a disaster strikes.  When it does, it needs to scale very quickly when people start asking for help.  In other words, you need scale-to-zero and a fast way to build and iterate on this new feature.
```
Comment (jkeam) [inline workshop/content/lab3.1_ml_overview.md]: Maybe the word "need" is a bit aggressive.  Maybe something softer like, "this is the perfect use case for scale-to-zero".

File: workshop/content/lab3.1_ml_overview.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Enter OpenShift Serverless.  Here is the architecture:
+
+![Architecture](images/ml_architecture_flow.png)
+
+You will focus on building, debugging, and deploying the NLP Prediction Service.  You will also integrate the following important components:
+
+* [Lifeline](https://github.com/RedHatGov/serverless-workshop-code/tree/main/lifeline) - a React based web application
+* [Twilio](https://www.twilio.com/) - a communications API for building the phone/text user interaction
+
+Feel free to ask your instructor if you have any questions with this architecture.  Let's get started!
```
Comment (jkeam) [inline workshop/content/lab3.1_ml_overview.md]: Maybe change to "if you have any questions about this architecture".

Changing "with" to "about".

File: workshop/content/lab3.1_ml_overview.md (hunk 1 above)
Comment (theckang) [reply workshop/content/lab3.1_ml_overview.md]: Will do

File: workshop/content/lab3.1_ml_overview.md (hunk 2 above)
Comment (theckang) [reply workshop/content/lab3.1_ml_overview.md]: Agreed

**Review by jkeam — APPROVED** [review]
looks good to me

**Comment by jkeam** [comment]
For the outstanding TODO, will you be doing that in another PR, or leave this open for that addition?

**Comment by theckang** [comment]
On hold until we incorporate: https://github.com/RedHatGov/serverless-workshop-code/pull/5

**Comment by theckang** [comment]
Reopening this PR.  Since we're faking the incident service call, we just need some sort of UI confirmation when the user hits "submit info" on the lifeline app.

**Review by dudash — APPROVED** [review]

**Pull request merged**